packages/webview_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.3.9+2
+version: 0.3.9+3
```

Comment:
minor:

I guess we can look at it either as a bug fix (fix webview keyboard prior to Android N), or as a feature(support webview keyboard prior to Android N) as we never supported it before.
I'm assuming people read into patch number bumps as: I can keep doing everything I did before, but some bugs I've hit are fixed and minor bumps as: there's new stuff I can do now. In this case I'd opt for a minor version bump given that people couldn't use keyboard before, and how expected this feature is. (Also rephrase the changelog update accordingly)

Reply:
Makes sense. Done.